nilearn/connectome/connectivity_matrices.py

```diff
@@ -584,6 +616,13 @@ def inverse_transform(self, connectivities, diagonal=None):
 
         if self.kind == 'tangent':
             mean_sqrt = _map_eigenvalues(lambda x: np.sqrt(x), self.mean_)
+            idty = np.eye(self.mean_.shape[0])
+            connectivities = [mean_sqrt.dot(
+                displacement + idty).dot(mean_sqrt)
```

Comment:
Shouldn't one of the two  mean_sqrt be transposed ? (not sure that they are symmetric)